src/main/scala/se/lth/immun/TargetDeisotoper.scala

```diff
@@ -62,74 +167,64 @@ class TargetDeisotoper(val params:DinosaurParams) extends Timeable {
 	def getPatterns(
 			seeds:Seq[Int], 
 			hills:Array[Hill],
-			t:Target
+			t:Target,
+			hillsMzs:Array[Double],
+			hillsScanIndices:Array[(Int, Int)]
 	):Seq[IsotopePatternInds] = {
 		if (seeds.isEmpty) return Nil
 		val patterns = 
 			for {
 				seed <- seeds
-				isopatInds <- extendSeed(seed, overlapping(seed, hills), hills, t.z)
-			} yield (isopatInds)
+				isoPatInds <- extendSeed(seed, hills, t.z, hillsMzs, hillsScanIndices)
+			} yield (isoPatInds)
 		if (patterns.isEmpty) Nil
 		else 
 			patterns.filter(ip => ip.inds.head == ip.seed)
 	}
 	
 	
-	def extendSeed(seed:Int, inds:Seq[Int], hills:Array[Hill], z:Int):Option[IsotopePatternInds] = {
+	def extendSeed(seed:Int, hills:Array[Hill], z:Int, hillsMzs:Array[Double], hillsScanIndices:Array[(Int, Int)]):Option[IsotopePatternInds] = {
 		
-		val seedTot = hills(seed).total
-			
-		def extend2(dir:Int):Seq[Int] = {
-			var ii = inds.indexOf(seed)+dir
+		val seedCenterMz = hillsMzs(seed)
+		val seedErr2 = hills(seed).total.centerMzError * hills(seed).total.centerMzError
+		val massErrorSq = params.adv.deisoSigmas * params.adv.deisoSigmas * (
+					  2.5*seedErr2)
```

Comment:
`2.5`?

Reply:
I think it was to represent the term `iHill.total.centerMzError*iHill.total.centerMzError`, by assuming that it is smaller than `sHill.total.centerMzError*sHill.total.centerMzError * 1.5`, where the `1.5` was an arbitrary safety margin. This eleminates the need of recalculating the `massErrorSq` for each iHill, which was apparently computationally expensive.